server/public/index.html

```diff
@@ -19,6 +19,7 @@
       crossorigin="anonymous"
     />
     <link rel="stylesheet" href="/styles.css" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.js"></script>
```

Comment:
do we need this?

Reply:
there is conflict on index.js
i checked that we need to remain everything.
check together and merge it tomorrow

Reply:
No, I forgot to remove this when I was trying different things to get Yelp requests to work. I have pushed another commit that removes this.
Nice catch :)